### PR TITLE
add new developer corner

### PR DESCRIPTION
**Pull request by wkerzendorf**
Update the docs to have a resource for the team to put descriptions

### CODE REVIEW

File: docs/development/c_numba_transition.rst [hunk 1]
```diff
@@ -0,0 +1,5 @@
+******************
+C-Numba transition
+******************
+
+
```
Comment (wkerzendorf) [inline docs/development/c_numba_transition.rst:5]:
```suggestion
We did several comparisons 
```

File: docs/development/c_numba_transition.rst [hunk 2]
```diff
@@ -0,0 +1,5 @@
+******************
+C-Numba transition
+******************
+
```
Comment (afloers) [inline docs/development/c_numba_transition.rst:4]:
```suggestion

Packet-by-packet comparison for simple tardis model (tardis_configv1_verysimple.yml) 
[PR929](https://github.com/tardis-sn/tardis/pull/929)

**Things to do:**
* line scattering (complete)
* electron scattering (complete)
* virtual packets (complete)
* macroatom (complete)
* packet logging (last_packet_interaction)

There are 4 code versions. There is upstream/master there is numba_montecarlo both of which have different seeding strategies and will never be numerically equal (those we will summarize as main). There are two versions that have additional code that will allow logging - those we will call c_compare and numba_compare (and will call the compare branch).

* virtual - only compare - sica (complete)
* virtual - only compare - ddc10
* integrated - main only - sica
* integrated - main only - ddc10
* estimators - compare only -sica (complete)
* estimators - compare only -ddc10 (complete)
* iterations - (compare/main) -sica (complete)
* iterations - (compare/main) -ddc10
* Need to fix negative boundary distance issue for vpackets: specific case occurs when a packet reaches a boundary with a close line so distance_trace_line = 0 and distance_boundary can be negative (shell ID does not advance). The models that shall be run is low velocity Si/Ca and DDC10.

**Post-merge**

* Start writing unit tests for the numba version (in progress)
* Improve speed.
* Close-lines need fixing in the numba code (C version not correct)
* Port formal integrator to numba 
```

File: docs/index.rst [hunk 3]
```diff
@@ -96,6 +96,7 @@ Mission Statement
 
     CONTRIBUTING.md
     development/index
```
Comment (smithis7) [inline docs/index.rst:98]: I think you want to change this to ```development/developer_workflow```

**Review by smithis7 — CHANGES_REQUESTED** [review]
In `docs/index.rst` you should change `development/index` to `development/developer_workflow` as the file name was changed.

**Comment by marxwillia** [comment]
I think this can be closed since the C-numba transition is complete.

**Pull request closed without merging**